kolibri/plugins/coach/assets/src/state/actions/exam.js

```diff
@@ -1,4 +1,5 @@
 const CoreApp = require('kolibri');
+const pick = require('lodash/fp/pick');
```

Comment:
This is using the 'fp' (functional programming) version of pick, which behaves a little bit differently than 'lodash/pick'